platforms/software/build-init/src/main/java/org/gradle/buildinit/plugins/BuildInitPlugin.java

```diff
@@ -72,7 +72,12 @@ public void apply(Project project) {
     }
 
     private static Provider<Boolean> getCommentsProperty(Project project) {
-        return project.getProviders().gradleProperty(COMMENTS_PROPERTY).map(Boolean::parseBoolean);
+        return project.getProviders().gradleProperty(COMMENTS_PROPERTY).map(new Transformer<Boolean, String>() {
```

Comment:
❔ Why is this change required?

Reply:
It's to make the init task CC compatible. We still can't use lambdas in the core plugins.

Reply:
Then you could use a helper method instead of an anonymous class: `SerializableLambdas.transformer(Boolean::parseBoolean)`

Reply:
I now realize the helper is an internal API, and we probably want to limit the usage of internal APIs even in the core plugin implementations.

However, the current requirement for wrapping everything into anonymous classes seems excessive just to make things CC compatible. Especially so, in the cases like this.